pennylane/ops/qubit/state_preparation.py

```diff
@@ -211,6 +223,18 @@ def compute_decomposition(state: TensorLike, wires: WiresLike) -> list[Operator]
         """
         return [MottonenStatePreparation(state, wires)]
```

Comment:
We should mention in the docstring that we are using `Mottonen` for the the decomposition on the un-supported devices.